src/AWS.Deploy.Orchestration/OptionSettingHandler.cs

```diff
@@ -81,6 +81,9 @@ private void SetDeploymentBundleProperty(Recommendation recommendation, OptionSe
                 case "DockerExecutionDirectory":
                     recommendation.DeploymentBundle.DockerExecutionDirectory = value.ToString() ?? string.Empty;
                     break;
+                case Constants.Docker.DockerfileOptionId:
```

Comment:
for consistency, could you update the reference of the other deployment bundle Ids to reference a const?